packages/core/src/Stores/client-store.js

```diff
@@ -252,7 +252,7 @@ export default class ClientStore extends BaseStore {
         if (Object.keys(this.currencies_list).length > 0) {
             const keys = Object.keys(this.currencies_list);
             // Fix for edge case when logging out from crypto accounts causes Fiat list to be empty
-            if (this.currencies_list.Fiat.length < 1) return 'USD';
+            if (this.currencies_list[localize('Fiat')].length < 1) return 'USD';
```

Comment:
why do you add localize here?

Reply:
@kelly-binary the object key `Fiat` is localized in `currency.js` - so if we don't localize here it will break the crowdIn in-context translations

Reply:
@oskar-binary okay, LGTM 👍

Reply:
wait, why are we localizing an object key?